spec/models/update_certificate_notification_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe UpdateCertificateNotification, type: :model do
+  it 'is valid and persisted' do
+    upload_certificate_event = create(:update_certificate_notification)
+    expect(upload_certificate_event).to be_valid
+    expect(upload_certificate_event).to be_persisted
+  end
+
+  it 'is valid and notication_sent defaults to false' do
```

Comment:
could the first two tests here be merged? given they're creating same event and just checking different fields they don't need to be separate